test/greet-test.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
Read Test requirements carefully. The test should also include testing for what happens if a user passes in a non-string.